resources/function_help/json/to_dm

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "to_dm",
+  "type": "function",
+  "description": "Convert a coordinate to degree, minute.",
+  "arguments": [
+    {"arg":"float","description":"A coordinate in EPSG:4326."},
```

Comment:
This doesn't necessarily *have* to be EPSG 4326  - rather any geographic coordinate system should be allowed. Maybe re-phrase as "A latitude or longitude value". I'd also suggest renaming `float` to `coordinate`.